diff --git a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
index e8ddce1..a617bc8 100644
--- a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
+++ b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
@@ -840,8 +840,8 @@ -(void)startObserving {
 
 - (void)emitEventInternal:(NSNotification *)notification {
   if (self.hasListeners) {
-    self.callableJSModules = [RCTAppleHealthKit sharedJsModule];
-    [self.callableJSModules setBridge:self.bridge];
+    // Patched for current React Native.
+    // RCTEventEmitter manages callableJSModules internally.
     [self sendEventWithName:notification.name
                    body:notification.userInfo];
   }
